Data/ByteString/Lazy/ReadInt.hs

```diff
@@ -0,0 +1,286 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE MultiWayIf #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+#include "MachDeps.h"
```

Comment:
This inclusion is now redundant. (likewise in ReadNat)

Reply:
Thanks, taking it out.